elyra/cli/pipeline_app.py

```diff
@@ -56,8 +56,7 @@ def _get_runtime_display_name(runtime_config: Optional[str]) -> Optional[str]:
 
     try:
         schema_manager = SchemaManager.instance()
-        schema_name = _get_runtime_type(runtime_config)
-        schema = schema_manager.get_schema('runtimes', schema_name)
+        schema = schema_manager.get_schema('runtimes', runtime_config)
```

Comment:
The second parameter is the name of the schema...
```suggestion
        schema = schema_manager.get_schema('runtimes', _get_runtime_type(runtime_config))
```
or leave what was there previously.